bundles/org.openhab.binding.miele/src/main/resources/OH-INF/thing/fridge.xml

```diff
@@ -16,8 +16,14 @@
 		<channels>
 			<channel id="state" typeId="state"/>
 			<channel id="supercool" typeId="supercool"/>
-			<channel id="current" typeId="current"/>
-			<channel id="target" typeId="target"/>
+			<channel id="current" typeId="temperature">
+				<label>Current Temperature</label>
+				<description>Current temperature in the fridge</description>
+			</channel>
+			<channel id="target" typeId="temperature">
+				<label>Target Temperature</label>
+				<description>Target temperature to be reached by the fridge</description>
+			</channel>
```

Comment:
As already mentioned, I don't thing this is a good idea to have the same channel type. Having both will then lead to no need to redefine the label and description here.

Reply:
The description is needed as it's tailored for each appliance type. But you are right about label, so will reconsider.

Reply:
You convinced me to split into current and target for reuse of same label - fixed.